CONTRIBUTING.rst

```diff
@@ -15,7 +15,7 @@ Create your dev database
     $ export PGDATABASE=procrastinate PGHOST=localhost PGUSER=postgres
     $ createdb
 
-Installation for development
+Set up your development environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The `^` below need to be as long as the title :o The CI is failing because of it

Reply:
(It's the case in multiple locations, I let you look into it :))